File: Models/ViewModels/NotifyVM.cs
using GoWheels_WebAPI.Models.Entities;

namespace GoWheels_WebAPI.Models.ViewModels
{
    public class NotifyVM
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public string? Content { get; set; }
        public int BookingId { get; set; }
        public DateTime CreateOn { get; set; }
        public bool IsRead { get; set; }
        public UserVM User { get; set; } = null!;
    }
}

File: FE/src/Model/NotifyVM.js
import UserVM from "./UserVM";
export default class NotifyVM {
    constructor(data = {}) {
      this.id = data.id || 0;
      this.bookingId = data.bookingId || 0;
      this.title = data.title|| 0;
      this.content = data.content || '';
      this.createOn = data.createOn || 0;
      this.isRead = data.isRead || '';
      this.user = (data.user || []).map(user => new UserVM(user));
    }
  }
  

